lib/flatten-browserlist.js

```diff
@@ -135,7 +135,7 @@ function flatten (request, all_browsers) {
         // JS will forget about the .0 when reading any float that can
         // be represented as an integer from yaml, so let's try to
         // match a version of that form as a fallback
-        return browser.version == version || browser.version == version + '.0'
+        return browser.version === version || browser.version === version + '.0'
```

Comment:
Both `browser.version` and `version` are strings here?

Reply:
Yes, it seems like it.